dev-itpro/administration/administration-center-api.md

```diff
@@ -27,7 +27,99 @@ For more information about administrative capabilities, see [The Business Centra
+The [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)] API supports authentication using AAD Apps.
+
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. Register an application for [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)] in your Azure Active Directory tenant.
+
+    Follow the general guidelines at [Register your application with your Azure Active Directory tenant](/azure/active-directory/active-directory-app-registration).
+
+    When you add an application to an Azure AD tenant, you must specify the following information:
+
+    |Setting|Description|
+    |-------|-----------|
+    |Name|Specify a unique name for your application. |
+    |Supported account types| Select either <strong>Accounts in this organizational directory only (Microsoft only - Single tenant)</strong> or <strong>Accounts in any organizational directory (Any Azure AD directory - Multitenant)</strong>.|
+    |Redirect URI|Optional. You can grant consent from the Azure portal if left empty.|
+
+    When completed, an **Overview** displays in the portal for the new application.
+3. Create a client secret for the registered application as follows:
+
+    1. Select **Certificates & secrets** > **New client secret**.
+    2. Add a description, select a duration, and select **Add**.
+
+    > [!NOTE]
+    > Copy the secret's value for use in your client application code. This secret value is never displayed again after you leave this page.
+
+    For the latest guidelines about adding client secrets in Azure AD, see [Add credentials](/azure/active-directory/develop/quickstart-register-app#add-credentials) in the Azure documentation.
+
+  > [!NOTE]
+  > Copy the **Application (client) ID** of the registered app. You'll need this later. You can get this value from the **Overview** page.
+4. Grant the registered application **AdminCenter.ReadWrite.All** permission to the **Dynamics 365 [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)]** API as follows:
+
+    1. Select **API permissions** > **Add a permission** > **Microsoft APIs**.
+    2. Select **Dynamics 365 [!INCLUDE [prod_short](../developer/includes/prod_short.md)]**.
+    3. Select **Application permissions**, select **AdminCenter.ReadWrite.All**, then select **Add permissions**.
+  
+    > [!NOTE]
+    > If you intent to use the same AAD App with the Automation API and Business Central Web Services you can also grant **API.ReadWrite.All** and **Automation.ReadWrite.All** permissions. Learn more [here](automation-apis-using-s2s-authentication.md).
+5. (optional) Grant admin consent on each permission by selecting it in the list, then selecting **Grant admin consent for \<tenant name\>**. This step isn't required if you'll be granting consent from the [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)].
```

Comment:
and additionally it is possible to grant consent from this page only for your own current tenant (so single-tenant apps are fine, for multi-tenant apps and different tenants you need to do that from AAD portal or from new TAC UI)